packages/libp2p/src/circuit-relay/transport/reservation-store.ts

```diff
@@ -287,23 +289,44 @@ export class ReservationStore extends EventEmitter<ReservationStoreEvents> imple
     throw new Error(errMsg)
   }
 
+  async removeListeners (peerId: PeerId): Promise<void> {
+    const listenersToRemove: string[] = []
+
+    this.transportManager.getListeners().forEach((listeners) => {
+      listeners.getAddrs().forEach((addr) => {
+        if (addr.toString().includes(peerId.toString())) {
+          listenersToRemove.push(addr.toString())
+        }
+      })
+    })
+
+    for (const addr of listenersToRemove) {
+      await this.transportManager.remove(addr)
+      log('removed addr: %s', addr)
+    }
+  }
+
   /**
    * Remove listen relay
    */
-  #removeRelay (peerId: PeerId): void {
+  async #removeRelay (peerId: PeerId): Promise<void> {
     const existingReservation = this.reservations.get(peerId)
 
     if (existingReservation == null) {
       return
     }
 
-    log('connection to relay %p closed, removing reservation from local store', peerId)
+    log.trace('connection to relay %p closed, removing reservation from local store', peerId)
```

Comment:
I have seen Alex mention standards for jslibp2p logging (trace for happy path, log for important entry/announce things, error for errors) but do we have this in docs somewhere?

Reply:
Not to my knowledge, but in any event seems like an appropriate place to put a trace log given that rubric. 

May be worthwhile starting a discussion thread for that.